test-utils/runtime/src/system.rs

```diff
@@ -39,19 +38,94 @@ use sp_std::prelude::*;
 const NONCE_OF: &[u8] = b"nonce:";
 const BALANCE_OF: &[u8] = b"balance:";
 
-decl_module! {
-	pub struct Module<T: Config> for enum Call where origin: T::Origin {}
-}
+pub use self::pallet::*;
+
+#[frame_support::pallet]
+mod pallet {
+	use super::*;
+	use frame_support::pallet_prelude::*;
+
+	#[pallet::pallet]
+	#[pallet::generate_store(pub(super) trait Store)]
+	pub struct Pallet<T>(PhantomData<T>);
+
+	#[pallet::config]
+	pub trait Config: frame_system::Config {}
+
+	pub struct ExtrinsicDataPrefix;
+	impl StorageInstance for ExtrinsicDataPrefix {
+		fn pallet_prefix() -> &'static str {
+			"TestRuntime"
+		}
+		const STORAGE_PREFIX: &'static str = "ExtrinsicData";
+	}
+	pub type ExtrinsicData =
+		StorageMap<ExtrinsicDataPrefix, Blake2_128Concat, u32, Vec<u8>, ValueQuery>;
+
+	// The current block number being processed. Set by `execute_block`.
+	pub struct NumberPrefix;
+	impl StorageInstance for NumberPrefix {
+		fn pallet_prefix() -> &'static str {
+			"TestRuntime"
+		}
+		const STORAGE_PREFIX: &'static str = "Number";
+	}
+	pub type Number = StorageValue<NumberPrefix, BlockNumber, OptionQuery>;
+
+	pub struct ParentHashPrefix;
+	impl StorageInstance for ParentHashPrefix {
+		fn pallet_prefix() -> &'static str {
+			"TestRuntime"
+		}
+		const STORAGE_PREFIX: &'static str = "ParentHash";
+	}
+	pub type ParentHash = StorageValue<ParentHashPrefix, Hash, ValueQuery>;
+
+	pub struct NewAuthoritiesPrefix;
+	impl StorageInstance for NewAuthoritiesPrefix {
+		fn pallet_prefix() -> &'static str {
+			"TestRuntime"
+		}
+		const STORAGE_PREFIX: &'static str = "NewAuthorities";
+	}
+	pub type NewAuthorities = StorageValue<NewAuthoritiesPrefix, Vec<AuthorityId>, OptionQuery>;
+
+	pub struct StorageDigestPrefix;
+	impl StorageInstance for StorageDigestPrefix {
+		fn pallet_prefix() -> &'static str {
+			"TestRuntime"
+		}
+		const STORAGE_PREFIX: &'static str = "StorageDigest";
+	}
+	pub type StorageDigest = StorageValue<StorageDigestPrefix, Digest, OptionQuery>;
+
+	pub struct AuthoritiesPrefix;
+	impl StorageInstance for AuthoritiesPrefix {
+		fn pallet_prefix() -> &'static str {
+			"TestRuntime"
+		}
+		const STORAGE_PREFIX: &'static str = "Authorities";
+	}
+	pub type Authorities = StorageValue<AuthoritiesPrefix, Vec<AuthorityId>, ValueQuery>;
+
+	#[pallet::genesis_config]
+	pub struct GenesisConfig<T: Config> {
+		pub authorities: Vec<AuthorityId>,
+		pub marker: PhantomData<T>,
+	}
```

Comment:
```suggestion
	pub struct GenesisConfig {
		pub authorities: Vec<AuthorityId>,
	}
```

Just don't make it generic, if you don't need it.